### ISSUE TITLE
Algorithmic Reverb Node

### ISSUE THREAD

**Issue by ddiakopoulos** [ddiakopoulos/LabSound#26]
It would be nice to have a lightweight, non-convolutional reverb. CCRMA's NRev algorithm would be a good candidate.

**Comment by ddiakopoulos** [comment]
https://github.com/CLOUDS-Interactive-Documentary/RTCmix/blob/960820f898a0a1f9a649fcb6f1ce452856170d18/insts/jg/objlib/NRev.cpp

**Comment by meshula** [comment]
I was looking at some noise based reverbs but I'm not convinced about those, they don't sound right to my ear.

NRev on the other hand looks nice! Haven't heard it yet, but the description is very convincing

**Comment by ddiakopoulos** [comment]
Yeah, NRev is pretty ok. Unfortunately two of the most popular reverbs (MVerb and GVerb) that are pretty good are both GPL. Freeverb might be another option.
